src/humidity/v1-0/v1-0.toit

```diff
@@ -16,7 +16,7 @@ READ-INDEX ::= 0
 /**
 A service that provides humidity readings.
 */
-interface HumidityService:
+interface Client:
```

Comment:
I think this is a bit confusing, because we often want to implement the "service" interface on the provider side too. That is the primary reason we don't call this client in general.